docs/auth/microsoft/provider.md

```diff
@@ -8,21 +8,36 @@ description: Adding Microsoft Azure as an authentication provider in Backstage
 The Backstage `core-plugin-api` package comes with a Microsoft authentication
 provider that can authenticate users using Azure OAuth.
 
-## Create an App Registration on Azure
-
-To support Azure authentication, you must create an App Registration:
-
-1. Log in to the [Azure Portal](https://portal.azure.com/)
-2. Create an
-   [Active Directory Tenant](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/Overview),
-   if one does not yet exist
-3. Navigate to
-   [Azure Active Directory > App Registrations](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredApps)
-4. Register an application
-   - Name: Backstage (or your custom app name)
-   - Redirect URI: Web >
-     `http://localhost:7007/api/auth/microsoft/handler/frame`
-5. Navigate to **Certificates & secrets > New client secret** to create a secret
+## Configure App Registration on Azure
+
+Depending on how locked down your company is, you may need a directory administrator to do some or all of these instructions.
+
+Go to [Azure Portal > App registrations](https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps/ApplicationsListBlade) and find your existing app registration, or create a new one.
+If you have an existing App Registration for Backstage, use that rather than create a new one.
+
+On your app registration's overview page, add a new `Web` platform configuration, with the configuration:
+
+- **Redirect URI**: `http://your-backstage.com/api/auth/microsoft/handler/frame` (for local dev, typically `http://localhost:7007/api/auth/microsoft/handler/frame`)
+- **Front-channel logout Url**: blank
+- **Implicit grant and hybrid flows**: All unchecked
+
+On the **API permissions** tab, click on `Add Permission`, then add the following `Delegated` permission for the `Microsoft Graph` API.
+
+- `email`
+- `offline_access`
+- `openid`
+- `profile`
+- `User.Read`
+
+Your company may require you to grant [admin consent](https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/user-admin-consent-overview) for these permissions.
+Even if your company doesn't require admin consent, you may wish to do so as it means users don't need to individually consent the first time they access backstage.
+To grant admin consent, a directory admin will need to come to this page and click on the **Grant admin consent for COMPANY NAME** button.
+
+![App Registration Permissions](permissions.png)
```

Comment:
```suggestion
![App Registration Permissions](./permissions.png)
```
When I use the rich preview this is a broken image for me, wondering if you need the `./` as suggested above

Reply:
Never mind, looks fine when I view it in your branch 👍